embedded-kafka/README.adoc

```diff
@@ -39,6 +43,11 @@ By default, to your projects `target` folder. You can configure binding using pr
 * `embedded.zookeeper.fileSystemBind.txnLogsFolder` `(default : target/embedded-zk-txn-logs)`
 * `embedded.kafka.fileSystemBind.enabled` `(true|false, default is false)`
 * `embedded.kafka.fileSystemBind.dataFolder` `(default : target/embedded-kafka-data)`
+* `embedded.kafka.toxiproxy.brokerList`
+* `embedded.kafka.toxiproxy.saslPlaintext.brokerList`
+* Bean `ToxiproxyContainer.ContainerProxy kafkaPlainTextContainerProxy`
+* Bean `ToxiproxyContainer.ContainerProxy kafkaSaslContainerProxy`
+* Bean `NetworkTestOperations kafkaNetworkTestOperations`
```

Comment:
I was thinking about deprecating `NetworkTestOperations` and instead provide users with `ToxiproxyContainer.ContainerProxy` beans instead. So that there was only one API available. What do you think?

Reply:
Ok, will remove